src/components/GeneralSettingsSection.vue

```diff
@@ -205,7 +205,7 @@
 						{{ t('backup', 'Request the creation of a new restoring point now') }}
 					</h3>
 					<div v-if="settings.restoringPointRequested" class="backup-settings__actions__action__info">
-						{{ t('backup', 'The creation of a restoring point as been requested and will be initiated soon.') }}
+						{{ t('backup', 'The creation of a restoring point has been requested and will be initiated soon.') }}
```

Comment:
This is the only spot where this needs to be fixed. The other languages/translations will pick it up automatically once it's fixed here. :)